tests/validators/answers/test_number_answer_validator.py

```diff
@@ -80,6 +80,107 @@ def test_are_decimal_places_valid():
     }
 
 
+def test_maximum_set_as_string():
+    answer = {
```

Comment:
for absolute clarity I might call this function
`test_invalid_maximum_value_is_string` and similar for the ones below